### ISSUE TITLE
Parser output seems to use 0-based line numbering for orchestras, 1-based for CSDs

### ISSUE THREAD

**Issue by nwhetsell** [csound/csound#546]
If I syntax-check this orchestra

```
prints
"hello, world"
```

by running in a Terminal

```sh
csound --syntax-check-only --orc Test.orc
```

then Csound reports the error at line&nbsp;1:

```
error: syntax error, unexpected STRING_TOKEN, expecting $end  (token ""hello, world"") from file Test.orc (1)
 line 1:
>>>"hello, world" <<<
```

This suggests 0-based line numbering. The output is the same whether the value of a `--csd-line-nums` flag is 0 or 1.

If I instead syntax-check this CSD

```
<CsoundSynthesizer>
<CsInstruments>
prints
"hello, world"
</CsInstruments>
</CsoundSynthesizer>
```

by running in a Terminal

```sh
csound --syntax-check-only Test.csd
```

then Csound reports the error at line&nbsp;4:

```
error: syntax error, unexpected STRING_TOKEN, expecting $end  (token ""hello, world"") from file Test.csd (1)
 line 4:
>>>"hello, world" <<<
```

This suggests 1-based line numbering. The output is the same whether the value of a `--csd-line-nums` flag is 0 or 1.

I’d expect reported line numbers to use either 0- or 1-based numbering for all files.

**Comment by jpffitch** [comment]
Think this is a problem in --syntax-check-only 
Actually I did not know this existed so wonder how it works

**Comment by nwhetsell** [comment]
It doesn’t look that way on my end. If I run `csound Test.orc Test.sco` with orchestra

```
//
//
prints
"hello, world"
```

and score

```
e
```

then Csound reports the error in line&nbsp;3:

```
virtual_keyboard real time MIDI plugin for Csound
0dBFS level = 32768.0
Csound version 6.06 (double samples) Oct 29 2015
libsndfile-1.0.25
orchname:  Test.orc
rtaudio: PortAudio module enabled ... 
using callback interface
rtmidi: PortMIDI module enabled

error: syntax error, unexpected STRING_TOKEN, expecting $end  (token ""hello, world"") from file Test.orc (1)
 line 3:
>>>"hello, world" <<<
```

If I run `csound Test.csd` with

```
<CsoundSynthesizer>
<CsInstruments>
prints
"hello, world"
</CsInstruments>
<CsScore>
e
</CsScore>
</CsoundSynthesizer>
```

then Csound reports the error in line&nbsp;4:

```
virtual_keyboard real time MIDI plugin for Csound
0dBFS level = 32768.0
Csound version 6.06 (double samples) Oct 29 2015
libsndfile-1.0.25
UnifiedCSD:  Test.csd
STARTING FILE
Creating orchestra
Creating score
rtaudio: PortAudio module enabled ... 
using callback interface
rtmidi: PortMIDI module enabled

error: syntax error, unexpected STRING_TOKEN, expecting $end  (token ""hello, world"") from file Test.csd (1)
 line 4:
>>>"hello, world" <<<
```

**Comment by jpffitch** [comment]
OK then it is in orc./sco format

**Comment by jpffitch** [comment]
Can you check vetsuon in git?  I think it is right for orc files and still wrong for sco files

**Comment by nwhetsell** [comment]
I can confirm that, with commit 9588534a227063d380061c12c1fa95cab4d55fd0, running `csound Test.orc Test.sco` with orchestra

```
//
//
prints
"hello, world"
```

and score

```
e
```

reports the error in line&nbsp;4, which agrees with the behavior for CSD files. I haven’t tested score files.

**Comment by nwhetsell** [comment]
This issue is again reproducible in Csound at commit cd4be5f71cb7b18ecb51e338f0b680676662b26f (and likely earlier). Running

```sh
csound --orc test.orc
```

with orchestra

```csound
//
//
prints
"hello, world"
```

results in

```
error: syntax error, unexpected STRING_TOKEN, expecting $end  (token """) from file … (1)
 line 2:
>>> <<<
```

Running

```sh
csound test.csd
```

with CSD

```csound-csd
<CsoundSynthesizer>
<CsInstruments>
prints
"hello, world"
</CsInstruments>
<CsScore>
e
</CsScore>
</CsoundSynthesizer>
```

results in

```
error: syntax error, unexpected STRING_TOKEN, expecting $end  (token """) from file … (1)
 line 3:
>>> <<<
```

**Comment by jpffitch** [comment]
Looks unexpected to me; or are you complaining about the line count

**Comment by nwhetsell** [comment]
I am pointing out (not complaining) that, in Csound at commit cd4be5f71cb7b18ecb51e338f0b680676662b26f, “Parser output seems to use 0-based line numbering for orchestras, 1-based for CSDs”. This issue was addressed in commit 9588534a227063d380061c12c1fa95cab4d55fd0 but is again reproducible.
